Add dev container configuration

r-a303931 opened the pull request:
Add dev container to allow developing around espeak-ng temporarily

This is a stop-gap solution until a more portable version of espeak-ng can be used

r-a303931 committed:
Add dev container configuration

Add dev container to allow developing around espeak-ng temporarily

This is a stop-gap solution until a more portable version of espeak-ng
can be used

chrisvire reviewed (changes requested):
Please add some documentation to main README.md  about how to use devcontainers

r-a303931 committed:
Add documentation and ffmpeg

Add documentation on opening the folder in VS Code dev containers

Modify Dockerfile to add ffmpeg to allow MP3 to WAV conversions

chrisvire reviewed (approved):
Looks good.  Update as you make improvements (like having default launch.json that can be referenced in dev container with some sample data.
